OpenWeather/App/Search/SearchScreen.swift

```diff
@@ -43,16 +43,24 @@ struct SearchScreen: View {
                 LocationResultItem(name: location.name)
             }
         }
-        .searchable(text: $searchText, prompt: "Search by name or zipcode")
+        .searchable(
+            text: Binding<String>(
+                get: { viewModel.searchText },
+                set: { viewModel.searchText = $0 }
+            ),
```

Comment:
We should be able to just say `$viewModel.searchText`.

Reply:
Oh never knew I can do this .. done.